### PR TITLE
[tools] (Sample Inclusion) Updating midprocessor to merge Script Lab snippets

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR changes the midprocessor to merge snippets from Script Lab that are mapped to the same API. The intent is that these snippets should be part of the same sample and should look like that in the docs. Now, such snippets will be in the same code block and not have a redundant link to the base sample.

### CODE REVIEW

**Comment by AlexJerabek** [comment]
@alison-mk, @samantharamon, @ElizabethSamuel-MSFT: If folks could double-check their areas, that would be great. This PR should just remove the redundant links and code block formatting. There are a couple changes from office-js-snippets and DefinitelyTyped that made it in here too, just FYI.

**Review by samantharamon — APPROVED** [review]
Formatting looks good in the Outlook snippets. Thanks!

**Review by alison-mk — APPROVED** [review]
Cool! Excel snippets look good.

File: docs/docs-ref-autogen/word/word/word.body.yml [hunk 1]
```diff
@@ -1356,12 +1352,8 @@ methods:
         await context.sync();
       });
 
-      ```
-
-      ```typescript
 
-      // Link to full sample:
-      https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/25-paragraph/get-word-count.yaml
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/word/word/word.body.yml:1364]: This snippet isn't from the same source as the other two. Can we collapse only the ones from the same snippet yaml file?

Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/word/word/word.body.yml:1364]: Internal task created to track more comprehensive resolution

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Word verified and looks good. Thx

**Pull request merged**